docs/source/blocks/block-style-wrapper.md

```diff
@@ -236,6 +236,83 @@ You need to manually add the above code in your view component block code to ben
 The styles in the block edit component are injected automatically into the blocks engine editor wrappers, so you don't have to take any action.
 ```
 
+## Nested custom CSS properties
+
+Given this block enhancer:
```

Comment:
Let's add an introduction to give purpose and context for why someone would want or not want to do this.

```suggestion
This section describes how to work with nested custom CSS properties.
You can inject custom CSS properties in a nested manner.
You can also avoid some nesting, where useful.


(inject-nested-custom-css-properties)=

### Inject nested custom CSS properties

Given this block enhancer:
```